lib/RexStan.php

```diff
@@ -34,6 +34,8 @@ public static function runFromCli(&$exitCode, ?string $path, ?int $level, ?strin
             $cmd .= ' --level '. $level;
         }
 
+        $cmd .= ' 2>&1';
```

Comment:
redirect errors into std-out, so the output for the rex console command will match 1:1 what PHPStan reports (and we do not separate stdout/stderr in separate blocks)